sgv2-docsapi/src/main/java/io/stargate/sgv2/docsapi/service/util/DocsApiUtils.java

```diff
@@ -15,14 +15,123 @@
  */
 package io.stargate.sgv2.docsapi.service.util;
 
+import com.google.common.base.Strings;
+import io.stargate.sgv2.docsapi.api.exception.ErrorCode;
+import io.stargate.sgv2.docsapi.api.exception.ErrorCodeRuntimeException;
 import io.stargate.sgv2.docsapi.service.query.DocsApiConstants;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
 
 public final class DocsApiUtils {
```

Comment:
Same here, the original code has more methods.

Reply:
The rest can be handled as part of the #1725 

Just not sure now how to handle the `DocsApiConstants` and the static usage here. Maybe in #1725 we are making this not static anymore..

Reply:
Maybe we can make it non-static immediately.. Better sooner than later..

Reply:
Kept is static, removed references to the constants..